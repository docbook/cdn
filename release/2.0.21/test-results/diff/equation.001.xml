<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <figure deltaxml:deltaV2="A" id="R.1.2" class="equation-wrapper">
            <div class="equation">
               <div id="R.1.2.2" class="mediaobject">
                  <img src="../graphics/emc2.png" alt="E=mc^2"/>
               </div>
            </div>
            <div class="title">Equation 1. First Equation</div>
         </figure><div deltaxml:deltaV2="B" class="content">
            <figure id="R.1.2" class="equation-wrapper">
               <div class="equation">
                  <div id="R.1.2.2" class="mediaobject">
                     <img src="../graphics/emc2.png" alt="E=mc^2"/>
                  </div>
               </div>
               <div class="title">Equation 1. First Equation</div>
            </figure>
            <p>An informal equation:</p>
            <figure id="R.1.4" class="informalequation-wrapper">
               <div class="informalequation">
                  <div id="R.1.4.1" class="mediaobject">
                     <img src="../graphics/emc2.png" alt="E=mc^2"/>
                  </div>
               </div>
            </figure>
            <p>And an inline one: <span class="inlineequation">
                  <span class="mathphrase">E=mc<sup>2</sup>
                  </span>
               </span>.
</p>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">An informal equation:</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><figure deltaxml:deltaV2="A" id="R.1.4" class="informalequation-wrapper">
            <div class="informalequation">
               <div id="R.1.4.1" class="mediaobject">
                  <img src="../graphics/emc2.png" alt="E=mc^2"/>
               </div>
            </div>
         </figure><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">And an inline one: <span class="inlineequation">
               <span class="mathphrase">E=mc<sup>2</sup>
               </span>
            </span>.
</p>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>