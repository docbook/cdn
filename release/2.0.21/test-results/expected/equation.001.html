<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><figure id="R.1.2" class="equation-wrapper"><div class="equation"><div id="R.1.2.2" class="mediaobject"><img src="../graphics/emc2.png" alt="E=mc^2" /></div></div><div class="title">Equation 1. First Equation</div></figure><p>An informal equation:</p><figure id="R.1.4" class="informalequation-wrapper"><div class="informalequation"><div id="R.1.4.1" class="mediaobject"><img src="../graphics/emc2.png" alt="E=mc^2" /></div></div></figure><p>And an inline one: <span class="inlineequation"><span class="mathphrase">E=mc<sup>2</sup></span></span>.
</p></article><script src="../../resources/js/prism.js"></script></body></html>